### ISSUE TITLE
Blocked on using AFNetworking uploadTaskWithStreamedRequest

### ISSUE THREAD

**Issue by venkatkongara** [Microsoft/WinObjC#2010]
As NSURLSessionUploadTask.mm has not implemented NSURLSessionUploadTask: method, I was not able to use the below code. It looks like the post method below is dependent internally in AFNetworking(third party OBJ-C SDK) on NSURLSessionUploadTask: method. Upon deploying the app and using the below method throws an exception in runtime.
I am using 10.0.0.14393 SDK and 2015 visual studio, AFNetworking 3.1.0, WINOBJC released on FEB1st
```objective-c
NSURLSessionTask*` newTask = POST:@"post"
            parameters:@{
                @"key" : @"value"
            }
            constructingBodyWithBlock:body
            progress:^(NSProgress* _Nonnull downloadProgress) {
                [self _reloadTaskProgress:newTask progress:downloadProgress];
            }
            success:^(NSURLSessionDataTask* _Nonnull task, id _Nullable responseObject) {
                [self _setTaskCompleted:newTask];

                NSData* responseData = [NSKeyedArchiver archivedDataWithRootObject:responseObject];
                [self _printOutput:@"POST Task completed with %d bytes:\n%@, %@\n", responseData.length, task.response, task.error];
                [self _printOutput:@"Reponse Object:\n%@\n", responseObject];

            }
            failure:^(NSURLSessionDataTask* _Nonnull task, NSError* _Nullable error) {
                [self _printOutput:@"Task Failed:\n%@\n", error];
            }];
```

**Comment by rajsesh-msft** [comment]
@venkatkongara, this is known issue, we didn't implement post/put.

**Comment by rajsesh-msft** [comment]
Duplicate of #941
